src/components/nowPlayingBar/nowPlayingBar.js

```diff
@@ -539,19 +539,27 @@ function updateNowPlayingInfo(state) {
             if (textLines[1].text) {
                 const text = document.createElement('a');
                 text.innerText = textLines[1].text;
+                documentTitle = text.innerText;
```

Comment:
This does currently show `Year - Movie Name` for movies and `Episode Number - Episode Name - Series` for episodes. Not sure if this is how we should go about it, maybe someone else can give suggestions.